pontoon/base/models.py

```diff
@@ -290,12 +290,26 @@ def serialized_notifications(self):
         is_comment = False
 
         if hasattr(notification.actor, "slug"):
-            actor = {
-                "anchor": notification.actor.name,
-                "url": reverse(
-                    "pontoon.projects.project", kwargs={"slug": notification.actor.slug}
-                ),
-            }
+            if "new string" in notification.verb and self.profile.custom_homepage:
```

Comment:
As you hinted at in the description, I'd remove the check for `self.profile.custom_homepage` here, and also in the other frontend below, of course.

The locale-agnostic translate view already handles the case where `custom_homepage` isn't set and redirects to the exact same URL we generate in the `else`-block, which you could then drop.

Reply:
One more thing I forgot to mention - it'd be slightly more reliable if we detected new string notifications with a regex match on the notification verb.

Reply:
Can do. The reason why I didn't do this initially is that if `custom_homepage` isn't set, locale-agnostic goes to the HTTP header's first `Accept-Language` locale. Now, if the user wants to get from there to whatever they consider the "right" locale, they need to 1. click on the current locale, 2. select another locale, 3. select the project where they were. That seems like it makes the cost of being wrong kinda high here, no?

Re: string matching, do I understand right that you'd like for the test condition to be stricter by requiring a closer match?

Reply:
While the price might be hard, very few people will need to pay for it. We have a localizer though that translates two locales (`dsb` and `hsb`). For him the current behaviour might actually be better.

@flodolo What's your take?

(Note that one day locales will be [easier to switch](https://bugzilla.mozilla.org/show_bug.cgi?id=1578235). 😄 We used to have a locale menu in the old Translate view, but then people were requesting an easier access to the dashboards.)

---

Re string matching, yes, that's what I'd like, but I'll leave the decision to you. It would be better if you could match `updated with {} new string` more closely, but as we discussed, this is only a temporary solution anyways. Going forward, we should group notifications properly with a dedicated field.

Reply:
I kind of like the idea of checking the custom homepage first. I'm afraid that we have a bunch of localizers translating content and using a different accept-languages in the browser.

Reply:
OK, let's keep the logic here!